ci/.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    permissions:
+        contents: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.18'
```

Comment:
It would be better to add a simple instruction to README where we should update on bumping Go version.
https://github.com/pion/.goassets/blob/165230aa523337c6c6e4c50d169198976bb43acf/ci/.github/workflows/test.yaml#L29
https://github.com/pion/.goassets/blob/165230aa523337c6c6e4c50d169198976bb43acf/ci/.github/workflows/test.yaml#L92
https://github.com/pion/.goassets/blob/165230aa523337c6c6e4c50d169198976bb43acf/ci/.github/workflows/test.yaml#L148

Reply:
Maybe we should automate it.

Reply:
Please add  ` # auto-update/latest-go-version` comment to this line

Reply:
Done! Love it when a plan comes together.